receiver/elasticsearchreceiver/metadata.yaml

```diff
@@ -1 +1,373 @@
 name: elasticsearchreceiver
+
+attributes:
+  elasticsearch.cluster.name:
+    description: The name of the elasticsearch cluster.
+  elasticsearch.node.name:
+    description: The name of the elasticsearch node.
+  cache_name:
+    description: The name of cache.
+    enum:
+    - fielddata
+    - query
+  fs_direction:
+    value: direction
+    description: The direction of filesystem IO.
+    enum:
+    - read
+    - write
+  segment:
+    description: The segment of JVM memory.
+    enum:
+    - heap
+    - nonheap
+  generation:
+    description: The generation on which garbage collection was performed.
+    enum:
+    - young
+    - old
+  memory_pool_name:
+    value: pool
+    description: The name of the JVM memory pool
+  disk_usage_state:
+    value: state
+    description: The state of a section of space on disk.
+    enum:
+    - used
+    - free
+  direction:
+    description: The direction of network data.
+    enum:
+    - received
+    - sent
+  document_state:
+    value: state
+    description: The state of the document.
+    enum:
+    - active
+    - deleted
+  shard_state:
+    value: state
+    description: The state of the shard.
+    enum:
+    - active
+    - relocating
+    - initializing
+    - unassigned
+  operation:
+    value: operation
+    description: The type of operation.
+    enum:
+    - index
+    - delete
+    - get
+    - query
+    - fetch
+    - scroll
+    - suggest
+    - merge
+    - refresh
+    - flush
+    - warmer
+  thread_pool_name:
+    description: The name of the thread pool.
+  thread_state:
+    value: state
+    description: The state of the thread.
+    enum:
+    - active
+    - idle
+  task_state:
+    value: state
+    description: The state of the task.
+    enum:
+    - rejected
+    - completed
+  health_status:
+    value: status
+    description: The health status of the cluster.
+    enum:
+    - green
+    - yellow
+    - red
+metrics:
+  # these metrics are from /_nodes/stats, and are node level metrics
+  elasticsearch.node.cache.memory.usage:
+    description: The size in bytes of the cache.
+    unit: By
+    sum:
+      monotonic: false
+      aggregation: cumulative
+      value_type: int
+    attributes: [cache_name]
+    enabled: true
+  elasticsearch.node.cache.evictions:
+    description: The number of evictions from the cache.
+    unit: "{evictions}"
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+    attributes: [cache_name]
+    enabled: true
+  elasticsearch.node.jvm.gc.classes.loaded:
```

Comment:
@jsuereth I'd appreciate it if you could take a look at these JVM metrics and make sure that these make sense here, or if there's another way I should be doing this.